imapsync/etc/e-smith/events/actions/nethserver-imapsync-systemdRun

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/perl
+
+#
+# Copyright (C) 2020 Nethesis S.r.l.
+# http://www.nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+use esmith::ConfigDB;
+use esmith::event;
+use strict;
+
+my $event = shift;
+my $name = shift;
+my $errors = 0;
+
+my $idb = esmith::ConfigDB->open_ro('imapsync') || die("Could not open imapsync database \n");
```

Comment:
Attention, you've created a database named `getmail` (see the empty directory above in this PR), I guess it should be `imapsync` as stated here :)

Reply:
thank for the pointer, however I am not sure my code is workable, still a draft

Reply:
done